Refactor knative related helm charts (#3187)
kyma-project/kyma: resources/knative-serving/README.md

# Knative Serving

## Overview

> This chart packs the [knative-serving](https://github.com/knative/docs/tree/master/docs/serving) release files.

Reviewer: ```suggestion
This chart includes [knative-serving](https://github.com/knative/docs/tree/master/docs/serving) release files.
```

 * Every CRD has the `helm.sh/hook: crd-install` annotation set. This forces Helm to install the CRDs before other resources.
 * The duplicate of the `images.caching.internal.knative.dev` CRD is removed from the serving release.
 * The `config-domain` is made configurable by specifying the `.Values.domainName` as the helm template.

>  * The creation of `knative-serving` Namespace has been removed as this will taken care by installation process.

Reviewer: ```suggestion
 * The `knative-serving` Namespace is no longer created. This happens during the installation process.
```
